ulib/prims.fst

```diff
@@ -731,4 +731,4 @@ let labeled (r: range) (msg: string) (b: Type) : Type = b
 (** THIS IS MEANT TO BE KEPT IN SYNC WITH FStar.CheckedFiles.fs
     Incrementing this forces all .checked files to be invalidated *)
 irreducible
-let __cache_version_number__ = 37
+let __cache_version_number__ = 38
```

Comment:
This will need to be bumped up (since the Tm_match residual comp change went it with the same version number).